prebid-mobile/modules/rendering/android-sdk-integration-gam.md

```diff
@@ -53,14 +37,19 @@ allprojects {
 
 App module build.gradle:
 
-```
+```text
 implementation('org.prebid:prebid-mobile-sdk-gam-event-handlers:x.x.x')
 ```
 
-## Banner API
+## AdUnit-Specific instructions
 
-To integrate the banner ad you need to implement three easy steps:
+This section covers integration details for different ad formats. In each scenario, you'll be asked for a `configId` - this is a key worked out with your Prebid Server provider. It's used at runtime to pull in the bidders and parameters specific to this adunit. Depending on your Prebid Server partner, it may be a UUID or constructed out of parts like an account number and adunit name.
+
+### Banners
 
+#### Display Banners
+
+To integrate the banner ad you need to implement three easy steps:
 
 ```kotlin
```

Comment:
For the future adjustments. We should use the same approach for creating the code snippets over the whole documentation. 

In this particular case, the code snippet presents only some piece of code, but in most other cases snippets alsow include the code of the function which holds this code. Usually, we take all code snippets from the Demo Applications. So it shouldn't be a problem to add the whole function in the same way as it exists in the Demo App.

Reply:
Sounds good to me, but understanding the code is more than I'm willing to do here. We'll need a volunteer to deal with this part.